### PR TITLE
add spacing between My Wallet Accounts and account name

### PR DESCRIPTION
**Pull request by rickycodes**
before:
image

after:
image

### CODE REVIEW

**Review by danjm — DISMISSED** [review]
looks good

File: ui/app/pages/settings/settings.component.js [hunk 1]
```diff
@@ -136,7 +136,9 @@ class SettingsPage extends PureComponent {
           className={c({ 'settings-page__subheader--link': initialBreadCrumbRoute })}
           onClick={() => initialBreadCrumbRoute && history.push(initialBreadCrumbRoute)}
         >{subheaderText}</div>
-        {breadCrumbTextKey && <div><span>{'> '}</span>{t(breadCrumbTextKey)}</div>}
+        {breadCrumbTextKey && <div
+          className={c({ 'settings-page__subheader--break': initialBreadCrumbRoute })}
```
Comment (Gudahtt) [inline ui/app/pages/settings/settings.component.js]: Should this be contingent upon `initialBreadCrumbRoute`? It looks like you're following the example set a couple lines up, which unfortunately isn't a very good example.

`classnames` expects arguments that are either strings, or objects with Boolean values. The example on line `136` really shouldn't be passing a string as the value - it should be using `isMyAccountsPage` instead (which is a Boolean, and which is `true` in every case that `initialBreadCrumbRoute` is truthy).

In this case, I think we want this style to always be applied? So you can just set it directly as the `className` property instead.

File: ui/app/pages/settings/settings.component.js [hunk 2]
```diff
@@ -136,7 +136,9 @@ class SettingsPage extends PureComponent {
...
+        ><span>{' > '}</span>{t(breadCrumbTextKey)}</div>}
         {isAddressEntryPage && <div><span>{' > '}</span>{addressName}</div>}
```
Comment (Gudahtt) [inline ui/app/pages/settings/settings.component.js]: The same class might be useful to apply here as well. The address entry page appears to suffer from the same problem

File: ui/app/pages/settings/settings.component.js (hunk 1 above)
Comment (rickycodes) [reply ui/app/pages/settings/settings.component.js]: updated based on feedback

File: ui/app/pages/settings/settings.component.js (hunk 2 above)
Comment (rickycodes) [reply ui/app/pages/settings/settings.component.js]: updated based on feedback

**Review by Gudahtt — APPROVED** [review]
Looks good!

**Review by danjm — APPROVED** [review]

**Pull request merged**